### PR TITLE
[#8] api endpoints

### PR DESCRIPTION
**Pull request by arekmano**
- Created classifiers for tenant/landlord and problem categories
- Added new endpoints
- Updated docker build process for this container

### CODE REVIEW

File: src/nlp_service/models/question.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    def __init__(self, questionID, conversationID, answerString):
+        self.questionID = questionID
+        self.conversationID = conversationID
+        self.answerString = answerString
+        return
+
+
+class QuestionOutput(object):
+    """docstring for Question"""
```
Comment (lancelafontaine) [inline src/nlp_service/models/question.py]: Don't seem very useful for now

**Review by lancelafontaine — APPROVED** [review]
Looks good to me, but you should update your branch with changes from master.

**Comment by arekmano** [comment]
updated from master

**Comment by arekmano** [comment]
I'd like @Vynny, @mihaiqc or @Samuel-Campbell to take a look at the changes as this will impact their work.

File: src/nlp_service/services/introduction_parser.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    def classify(introduction):
+        personClass = 'unknown'
+        if introduction.personClassString == "test":
+            personClass = 'landlord'
+        else:
+            personClass = 'tenant'
+
+        return IntroductionOutput(
+            introduction.conversationID,
+            'Jackie Chan',
```
Comment (Vynny) [inline src/nlp_service/services/introduction_parser.py:16]: This should be the person's name

**Review by Vynny — CHANGES_REQUESTED** [review]

**Comment by Samuel-Campbell** [comment]
Only comment I have is that I notice you don't use encapsulation. Other than that looks fine

File: src/nlp_service/services/introduction_parser.py (hunk 2 above)
Comment (arekmano) [reply src/nlp_service/services/introduction_parser.py:16]: This functionality isn't implemented yet. It returns a static string until we figure it out.

**Comment by arekmano** [comment]
I didn't use encapsulation because it is easier to debug this way.

**Review by Samuel-Campbell — APPROVED** [review]

**Review by Vynny — APPROVED** [review]

**Pull request merged**